### PR TITLE
gofmt -w test/e2e/*

### PR DESCRIPTION
**Pull request by jbpratt**
# Changes

format test/e2e files

 #1481

changes can be reproduced with `gofmt -w test/e2e/*`

Signed-off-by: jbpratt <jbpratt78@gmail.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/clustertask/start_test.go [hunk 1]
```diff
@@ -1,4 +1,6 @@
+//go:build e2e
```
Comment (pradeepitm12) [inline test/e2e/clustertask/start_test.go:1]: [tektoncd/plumbing/](https://github.com/tektoncd/plumbing/blob/759f977c9f1b2afd2c88cafad4f23e944d8e60b2/tekton/images/test-runner/Dockerfile#L172) is still using GOLANGCI_VERSION 1.42.0
and `//go:build e2e` is supported by GOLANGCI_VERSION 1.44.0 and above 
ref: https://pkg.go.dev/cmd/go#hdr-Build_constraints

Comment (jbpratt) [reply test/e2e/clustertask/start_test.go:1]: As reference to this in documentation:
```
Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax. 
```

Comment (pradeepitm12) [reply test/e2e/clustertask/start_test.go:1]: Oh, I was wondering if ci would fail. Never mind  :+1:

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

**Review by vdemeester — APPROVED** [review]

**Pull request merged**